test/startservers.py

```diff
@@ -32,29 +32,18 @@ def run(self):
 processes = []
 
 
-def install(progs, race_detection):
-    cmd = "go install"
+def install(race_detection):
+    # Pass an empty BUILD_TIME flag to avoid constantly invalidating the build
+    # cache with new BUILD_TIMEs.
+    cmd = "make BUILD_TIME="
     if race_detection:
-        cmd = """go install -race"""
+        cmd = cmd + "GO_BUILD_FLAGS=-race"
```

Comment:
(The quotation marks, not because they are required, but because they might make things scream a little louder if someone forgets a space again later.)